packages/generator/src/edmx-parser/v4/edmx-parser.ts

```diff
@@ -31,7 +31,11 @@ export function parseComplexTypes(root): EdmxComplexType[] {
 }
 
 export function parseEnumTypes(root): EdmxEnumType[] {
-  return forceArray(root.EnumType);
+  const types: EdmxEnumType[] = forceArray(root.EnumType);
+  return types.map(edmxEnumType => ({
+    Name: edmxEnumType.Name,
+    Member: forceArray(edmxEnumType.Member)
```

Comment:
When there is only one member, the type of the `Member` is an object instead of an array.

Reply:
That is essentially the fix, right? ;)